.github/workflows/linux.yml

```diff
@@ -240,7 +240,9 @@ jobs:
         run: make wasi-libc
       - run: make gen-device -j4
```

Comment:
`make gen-device` shouldn't be necessary anymore with this PR?

Reply:
Should `gen-device` be moved over to the `device` package?